<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionDlg</class>
 <widget class="QDialog" name="optionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选项设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/option.ico</normaloff>:/option.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWebServerSet">
      <attribute name="title">
       <string>WEB服务器设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="hLayoutWebServerSet">
         <item>
          <widget class="QLabel" name="labelWebUrl">
           <property name="text">
            <string>服务器地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditWebUrl">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>请输入Web服务器地址</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxIsUsedVerifiCode">
         <property name="text">
          <string>是否使用验证码登陆</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutVerifiCodeSet">
         <item>
          <widget class="QLabel" name="labelVerifiCodeUrl">
           <property name="text">
            <string>验证码地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVerifiCodeUrl">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>请输入验证码地址</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOtherSet">
      <attribute name="title">
       <string>其他设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutBtn">
     <item>
      <widget class="QPushButton" name="pBtnDefault">
       <property name="text">
        <string>恢复默认设置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnSave">
       <property name="text">
        <string>保 存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnCancel">
       <property name="text">
        <string>取 消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelMsg">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
